modules/mono/utils/macos_utils.h

```diff
@@ -28,15 +28,6 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-#ifndef MONO_MACOS_UTILS_H
-#define MONO_MACOS_UTILS_H
```

Comment:
Same here, it is included in non-macOS specific `editor_internal_calls.cpp` C++ file.